From 1238688143cc435ec626f3aac7f718ce323b6433 Mon Sep 17 00:00:00 2001
From: Milian Wolff <mail@milianw.de>
Date: Tue, 31 Jan 2017 17:16:29 +0100
Subject: [PATCH] Fix compile: add missing QImage include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

/home/milian/projects/kf5/src/kde/pim/kdepim-apps-libs/kaddressbookgrantlee/src/printing/contactgrantleeprintobject.cpp:257:74: error: invalid use of incomplete type ‘class QImage’
---
 kaddressbookgrantlee/src/printing/contactgrantleeprintobject.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kaddressbookgrantlee/src/printing/contactgrantleeprintobject.cpp b/kaddressbookgrantlee/src/printing/contactgrantleeprintobject.cpp
index 9e6f757..baf4294 100644
--- a/kaddressbookgrantlee/src/printing/contactgrantleeprintobject.cpp
+++ b/kaddressbookgrantlee/src/printing/contactgrantleeprintobject.cpp
@@ -32,6 +32,7 @@
 #include <QVariant>
 #include <grantlee/metatype.h>
 #include <QLocale>
+#include <QImage>
 
 using namespace KAddressBookGrantlee;
 
-- 
2.10.2

